src/pages/components/filter-panel.mdx

```diff
@@ -0,0 +1,268 @@
+
+</Column>
+
+</Row>
+
+### Use case
+
+
+![A visual example of the Filter panel component in context of a page](../../images/component/filter-panel/filter-panel-usecase.jpg)
+  
+
+## Anatomy
+
+<Row>
+
+<Column colMd={4} colLg={6}>
+
+![A visual example of the anatomy of the Filter panel component](../../images/component/filter-panel/filter-panel-anatomy.jpg)
+
+</Column>
+
+</Row>
+
+<br />
+
+  
+1. **Children:** Optional container that allows components like tag group.
+2. **Header:** Customizeable header for the filter panel.
+3. **Category:** Customizable category header for a group of filters (required).
+4. **Filter item:** Customizable filter item, it can be a single or multi select (required).
+5. **Clear button:** Customizable button that deselects all selected filter items within the filter panel.
+
+
+## Selection methods
+Choosing the right filter selection method will improve usability and user efficiency. Filter panel supports several selection methods that are appropriate for different situations. Consider the data your users are looking at, what they are trying to achieve, and how they might intuitively narrow down the data.
+
+### Single selection
+Use a single-selection filter when the user can pick only one attribute to modify data results. Under the hood, single selection behaves like a radio button.
+
+<Row>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: single selection with no selected filter.](../../images/component/filter-panel/filter-panel-single-selection-example-not-selected.jpg)
+
+<Caption>Example of single selection with no filter item selected.</Caption>
+
+</Column>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: single selection with selected filter.](../../images/component/filter-panel/filter-panel-single-selection-example-selected.jpg)
+
+<Caption>Example of single selection with a filter item selected.</Caption>
+
+</Column>
+
+</Row>
+
+Single-selection supports multiple tiers of content. Second tier content is exposed after the user selects the parent filter, tiered content is indented below the parent filter.
+
+<Row>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: single selection hover state of a parent tier filter item.](../../images/component/filter-panel/filter-panel-single-selection-parent-tier.jpg)
+
+<Caption>Example of single selection hover state of a parent tier filter item.</Caption>
+
+</Column>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: parent tier filter item selected, with child tier items exposed.](../../images/component/filter-panel/filter-panel-single-selection-child-tier.jpg)
+
+<Caption>Example of parent tier filter item selected, with child tier items exposed.</Caption>
+
+</Column>
+
+</Row>
+
+### Multiselect
+Use a multiselect filter when the user has the option to pick more than one attribute to modify the data results. Under the hood, multiselects behave like checkboxes.
```

Comment:
Just want to double-check...Under the hood, `multiselects` behave like checkboxes. Should this be `multiselect` behaves like checkboxes?

Reply:
This is good as is, @JennySanchez.